lib/common/commonUtil.py

```diff
@@ -560,6 +562,104 @@ def deep_merge_dict(source_dict, destination_dict):
         return destination_dict
 
 
+class NetworkUtil(object):
+
+    @staticmethod
+    def download_file(output_dp, download_link):
+        logger.info("Prepare to download the file from link [%s]" % download_link)
+        response = requests.get(download_link, verify=False, stream=True)
+        download_fn = download_link.split("/")[-1]
+        if os.path.exists(output_dp) is False:
+            os.makedirs(output_dp)
+        download_fp = os.path.join(output_dp, download_fn)
+        try:
+            try:
+                total_len = int(response.headers['content-length'])
+            except:
+                total_len = None
+            with open(download_fp, 'wb') as fh:
+                for data in tqdm(response.iter_content(chunk_size=512 * 1024), total=total_len / (512 * 1024)):
+                    fh.write(data)
+            return download_fp
+        except Exception as e:
+            logger.error("ERROR: [%s]" % e.message)
+            return None
+
+    @staticmethod
+    def get_files_from_remote_url_folder(input_url, input_params=None):
+        return_dict = {}
```

Comment:
adding the comment here to describe the return object will be better.

(the name `get_files_...` looks like return a array.)